### PR TITLE
Fix readme for multi-user accounts and zone problems.

### PR DESCRIPTION
**Pull request by mberhault**

### CODE REVIEW

File: terraform/aws/variables.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ variable "aws_region" {
 
 # AWS availability zone. Make sure it exists for your account.
 variable "aws_availability_zone" {
-  default = "us-east-1a"
+  default = "us-east-1b"
```
Comment (bdarnell) [inline terraform/aws/variables.tf]: Why this change? Is the 1b zone more likely to exist than 1a? The "Make sure it exists for your account" comment needs some elaboration.

Comment (mberhault) [inline terraform/aws/variables.tf]: it doesn't exist on our shared account (bleh. I still don't understand why this is dynamic).
I put the variable under "check your setup" of the readme.
